testament/testament.nim

```diff
@@ -273,12 +273,14 @@ proc addResult(r: var TResults, test: TTest, target: TTarget,
                             expected = expected,
                             given = given)
   r.data.addf("$#\t$#\t$#\t$#", name, expected, given, $success)
+  template disp(msg) =
```

Comment:
Nitpick (don't have to change it), I use ``disp`` for "dispatch" and there is no conditional here. :-)